old_timesheet_migration/timesheet_migration.py

```diff
@@ -0,0 +1,233 @@
+
+
+def fill_ids(time_entries):
+    """Receives a data frame with all time entries without ids
+       Returns a data frame with ids."""
+
+    member_ids = {}
+    members_acronym = time_entries.member_acronym.unique()
+    for acronym in members_acronym:
+        email = {"email": acronym + "@certi.org.br"}
+        member_ids[acronym] = get_entity_id("Member", {"acronym": acronym}, email)
+
+    project_ids = {}
+    project_names = time_entries.project_name.unique()
+    for name in project_names:
+        project_ids[name] = get_entity_id("Project", {"name": name})
+
+    activity_ids = {}
+    activity_names = time_entries.activity_name.unique()
+    for name in activity_names:
+        activity_ids[name] = get_entity_id("Activity", {"name": name})
+
+    client_ids = {}
+    client_names = time_entries.client_name.unique()
+    for name in client_names:
+        client_ids[name] = get_entity_id("Client", {"name": name})
+
+    time_entries_with_ids = update_time_entries_ids(
+        time_entries, member_ids, project_ids, activity_ids, client_ids
+    )
+
+    return time_entries_with_ids
+
+
+def update_time_entries_ids(
+    time_entries, member_ids, project_ids, activity_ids, client_ids
+):
+    """Receives a dataframe that represents the time_entries without the ids
+        Returns with the correct ids, based on the dictionaries member_ids,
+        project_ids, activity_ids and client_ids"""
+    for index, row in time_entries.iterrows():
+        time_entries.loc[index, "member_id"] = member_ids[row[0]]
+        time_entries.loc[index, "project_id"] = project_ids[row[2]]
+        time_entries.loc[index, "activity_id"] = activity_ids[row[4]]
+        time_entries.loc[index, "client_id"] = client_ids[row[6]]
+    return time_entries
+
+
+def send_to_database(time_entries):
+    """Sends time entries to database
+       Return nothing."""
+    time_entries = time_entries.reset_index()
+    settings.db.table("time_entry").insert(
+        time_entries.drop(
+            ["index", "activity_name", "project_name", "client_name", "member_acronym"],
+            axis=1,
+        ).to_dict("records")
+    )
+
+
+def import_timesheets():
+    """Function that creat a timesheet migration to neodata of a path.
+       Returns nothing"""
+
+    header_time_entries = {
+        "member_acronym": "",
+        "member_id": "",
+        "project_name": "",
+        "project_id": "",
+        "activity_name": "",
+        "activity_id": "",
+        "client_name": "",
+        "client_id": "",
+        "start": "",
+        "end": "",
+    }
+    empty_time_entries = pd.DataFrame(columns=header_time_entries)
+
+    timesheets_path = join(
+        dirname(dirname(abspath(__file__))), "old_timesheet_migration", "timesheets"
+    )
+    os.chdir(timesheets_path)
+
+    for timesheet in glob.glob("*xlsx"):
+        year_semester = timesheet.split("_")[3].split(".")[0][:5]
+        path = timesheets_path + "/" + timesheet
```

Comment:
We should use os.join
```suggestion
        path = os.path.join(timesheets_path, timesheet)
```
This makes it usable for Windows, Linux, and Mac.